### PR TITLE
Amlandeep/add new field to sdk response

### PR DESCRIPTION
**Pull request by koko1123**
Closes: #1987

Add the new field from protobuf to flow-sdk to use in integration test

### CODE REVIEW

File: client/client.go [hunk 1]
```diff
@@ -588,5 +588,6 @@ func (c *Client) GetExecutionResultForBlockID(ctx context.Context, blockID flow.
 		BlockID:          flow.BytesToID(er.ExecutionResult.BlockId),
 		Chunks:           chunks,
 		ServiceEvents:    serviceEvents,
+		ExecutionDataId:  flow.BytesToID(er.ExecutionResult.ExecutionDataId),
```
Comment (sideninja) [inline client/client.go:591]:
```suggestion
		ExecutionDataID:  flow.BytesToID(er.ExecutionResult.ExecutionDataID),
```

File: execution_result.go [hunk 2]
```diff
@@ -7,6 +7,7 @@ type ExecutionResult struct {
 	BlockID          Identifier // commit of the current block
 	Chunks           []*Chunk
 	ServiceEvents    []*ServiceEvent
+	ExecutionDataId  Identifier
```
Comment (sideninja) [inline execution_result.go:10]:
```suggestion
	ExecutionDataID  Identifier
```

**Comment by sideninja** [comment]
@koko1123 what is the status of this? I plan to get a new release out in the next days and it would be nice to merge this.

**Comment by koko1123** [comment]
No longer need these changes

**Pull request closed without merging**